packages/docusaurus/src/client/templates/index.html.template.ejs

```diff
@@ -5,6 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="generator" content="Docusaurus">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <%if (htmlWebpackPlugin.options.noIndex) { %>
+      <meta name="robots" content="noindex" />
+    <% } %>
```

Comment:
this is the template for webpack dev server (not very clear I know 😅 )

we don't need noindex in dev